### PR TITLE
feat: Import ios/android BLE drivers from dev repo

### PR DESCRIPTION
**Pull request by D4ryl00**
Import iOS and Android BLE drivers from private dev repository.

### CODE REVIEW

File: go/internal/ble-driver/BleInterface_darwin.m [hunk 1]
```diff
@@ -20,57 +20,54 @@
...
+    os_log_error(OS_LOG_BLE, "🔴 Unhandled exception %{public}@", exception);
 }
 
 BleManager* getManager(void) {
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        os_log(OS_LOG_BLE, "getManager() initialize");
-        manager = [[BleManager alloc] initScannerAndAdvertiser];
+        os_log(OS_LOG_BLE, "🟢 getManager() initialize");
+        manager = [[[BleManager alloc] initScannerAndAdvertiser] autorelease];
```
Comment (aeddi) [inline go/internal/ble-driver/BleInterface_darwin.m:30]: If I remember correctly: `autorelease` is only useful if used in conjunction with an `autoreleasepool`. But I'm not sure.

File: go/internal/ble-driver/BleInterface_darwin.m [hunk 2]
```diff
@@ -20,57 +20,54 @@
...
-        [bDevice writeToCharacteristic:[NSMutableData dataWithData:cPayload] forCharacteristic:bDevice.writer withEOD:FALSE andBlock:^(NSError *error) {
-            blockError = [error copy];
-            dispatch_semaphore_signal(sema);
-        }];
-        dispatch_semaphore_wait(sema, DISPATCH_TIME_FOREVER);
-        dispatch_release(sema);
-        if (blockError == nil) {
-            return 1;
-        }
+        [bDevice writeToCharacteristic:[NSMutableData dataWithData:cPayload] forCharacteristic:bDevice.writer withEOD:FALSE tryL2cap:TRUE];
```
Comment (aeddi) [inline go/internal/ble-driver/BleInterface_darwin.m:62]: Why there is no error handling anymore on `writeToCharacteristic`?

File: js/android/app/src/main/java/tech/berty/gobridge/BertyNearbyDriver.java [hunk 3]
```diff
@@ -7,23 +7,23 @@
...
-import tech.berty.gobridge.proximitydriverssdk.lifecycle.UserRequestCallback;
-import tech.berty.gobridge.proximitydriverssdk.lifecycle.UserSearchCallback;
-import tech.berty.gobridge.proximitydriverssdk.model.Endpoint;
+import tech.berty.gobridge.nearbydriversdk.base.NearbyDriverSDK;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserAcceptCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserConnectionCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserMessageCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserRequestCallback;
+import tech.berty.gobridge.nearbydriversdk.lifecycle.UserSearchCallback;
+import tech.berty.gobridge.nearbydriversdk.model.Endpoint;
```
Comment (aeddi) [inline js/android/app/src/main/java/tech/berty/gobridge/BertyNearbyDriver.java:22]: Can you rename `nearbydriversdk` -> `nearbydriver` and `NearbyDriverSDK` -> `NearbyDriver`?
As it's not an SDK but actually a driver using the SDK ;)

File: js/packages/store/providerEffects.ts [hunk 4]
```diff
@@ -197,6 +197,11 @@ export const openingDaemon = async (
 				? [...bridgeOpts.cliArgs!, '--p2p.multipeer-connectivity=false']
 				: [...bridgeOpts.cliArgs!, '--p2p.multipeer-connectivity=true']
 
+		// set mc flag
```
Comment (aeddi) [inline js/packages/store/providerEffects.ts:200]:
```suggestion
		// set nearby flag
```

File: js/android/app/src/main/java/tech/berty/gobridge/BertyNearbyDriver.java (hunk 3 above)
Comment (aeddi) [reply js/android/app/src/main/java/tech/berty/gobridge/BertyNearbyDriver.java:22]: (Same things goes for all the imports, filenames, etc... below)

**Review by aeddi — CHANGES_REQUESTED** [review]

File: go/internal/ble-driver/BleInterface_darwin.m (hunk 1 above)
Comment (D4ryl00) [reply go/internal/ble-driver/BleInterface_darwin.m:30]: Here is the `autoreleasepool` :)
https://github.com/berty/berty/pull/3389/files/a253a9c3f05a92a056b0468e71c04ea0134ace1b#diff-c22f6876520e4ee313ba1188699ae63a388fcad1b2ef82f655015878946b62e2R38

Comment (aeddi) [reply go/internal/ble-driver/BleInterface_darwin.m:30]: Haha oops I missed it! :)
Alright 👍

**Pull request merged**